<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>depositwindow</class>
 <widget class="QWidget" name="depositwindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(91, 107, 128);</string>
  </property>
  <widget class="QPushButton" name="button_goToCredit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 69, 147);
image: url(:/new/prefix1/img/monetki.png);
background-color: rgb(48, 83, 129);
font: 500 11pt &quot;Ubuntu&quot;;
color: rgb(25, 37, 59);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_goToCalc">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>20</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(13, 69, 147);
image: url(:/new/prefix1/img/calc_icon.png);
background-color: rgb(48, 83, 129);
font: 500 11pt &quot;Ubuntu&quot;;
color: rgb(25, 37, 59);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>751</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(144, 183, 237);
background-color: rgb(91, 107, 128);</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>126</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(216, 220, 234);</string>
    </property>
    <property name="text">
     <string>start of term:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_term">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(216, 220, 234);
</string>
    </property>
    <property name="text">
     <string>deposit term:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_amount">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(216, 220, 234);
</string>
    </property>
    <property name="text">
     <string>deposit amount:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_interest">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(216, 220, 234);</string>
    </property>
    <property name="text">
     <string>interest rate:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkCap">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(216, 220, 234);
</string>
    </property>
    <property name="text">
     <string>Capitalization</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateStart">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>131</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(216, 220, 234);
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_calculateDeposit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>340</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(13, 69, 147);
color: rgb(216, 220, 234);
background-color: rgb(69, 122, 192);
font: 500 16pt &quot;Ubuntu&quot;;
</string>
    </property>
    <property name="text">
     <string>calculate</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textAccured">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>50</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(62, 72, 87);
color: rgb(216, 220, 234);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>50</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Ubuntu&quot;;
color: rgb(216, 220, 234);</string>
    </property>
    <property name="text">
     <string>accured interest:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textTax">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>110</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(62, 72, 87);
color: rgb(216, 220, 234);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>110</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Ubuntu&quot;;
color: rgb(216, 220, 234);</string>
    </property>
    <property name="text">
     <string>tax amount:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textFinal">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>170</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(62, 72, 87);
color: rgb(216, 220, 234);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>170</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;Ubuntu&quot;;
color: rgb(216, 220, 234);</string>
    </property>
    <property name="text">
     <string>final amount:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_payoutFreq">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(216, 220, 234);</string>
    </property>
    <property name="text">
     <string>payout frequency:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputInterest">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>170</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(62, 72, 87);
color: rgb(216, 220, 234);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputTerm">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>90</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(62, 72, 87);
color: rgb(216, 220, 234);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputAmount">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(62, 72, 87);
color: rgb(216, 220, 234);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboMonths">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>90</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 500 11pt &quot;Ubuntu&quot;;
color: rgb(216, 220, 234);
</string>
    </property>
    <item>
     <property name="text">
      <string>Days</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Months</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Years</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboPayout">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 500 11pt &quot;Ubuntu&quot;;
color: rgb(216, 220, 234);</string>
    </property>
    <item>
     <property name="text">
      <string>daily</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>weekly</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>monthly</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>quarterly</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>semi-annualy</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>yearly</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_tax">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(216, 220, 234);</string>
    </property>
    <property name="text">
     <string>tax rate:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputTax">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(62, 72, 87);
color: rgb(216, 220, 234);</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <zorder>widget</zorder>
  <zorder>button_goToCredit</zorder>
  <zorder>button_goToCalc</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
